tests/common-tests.h

```diff
@@ -172,6 +172,66 @@ std::ostream& operator<<(std::ostream& ostr, const std::vector<MyOptionalString>
     return ostr;
 }
 
+// The next classes are used for DDL testing
```

Comment:
This comment is really not very useful :-( I'd appreciate if it could describe at least a bit what's going on here.